commit a2a2aabdf535cb7f09dfa1c2d4e10566246b06a9
Author: Sergei Trofimovich <slyich@gmail.com>
Date:   Sun Feb 27 10:59:57 2022 +0000

    BUILD(positional-audio): Fix missing <memory> include
    
    Without the change the build fails on upcoming gcc-12 as:
    
        /build/mumble/plugins/gtav/gtav.cpp:13:13:
          error: 'unique_ptr' in namespace 'std' does not name a template type
           13 | static std::unique_ptr< Game > game;
              |             ^~~~~~~~~~
        /build/mumble/plugins/gtav/gtav.cpp:12:1:
          note: 'std::unique_ptr' is defined in header '<memory>';
            did you forget to '#include <memory>'?
           11 | #include <cstring>
          +++ |+#include <memory>
           12 |
    
    (cherry picked from commit 36398fb3b293fc889bbc38df29665359f5c2a0ca)

diff --git a/plugins/gtav/gtav.cpp b/plugins/gtav/gtav.cpp
index 468ff8922..829b1f544 100644
--- a/plugins/gtav/gtav.cpp
+++ b/plugins/gtav/gtav.cpp
@@ -9,6 +9,7 @@
 #include "mumble_positional_audio_utils.h"
 
 #include <cstring>
+#include <memory>
 
 static std::unique_ptr< Game > game;
 
